test/cluster-py/multi.test.py

```diff
@@ -0,0 +1,77 @@
+import sys
+import os
+from threading import Thread
+import time
+import yaml
+from lib.tarantool_server import TarantoolServer
+sys.path.append('../tarantool')
+from mesh_connection import MeshConnection
+from tarantool.const import (
+    SOCKET_TIMEOUT,
+    RECONNECT_DELAY,
+)
+from tarantool.error import NetworkError
+from tarantool.utils import ENCODING_DEFAULT
+
+REPLICA_N = 2
+
+
+def check_cluster(addrs):
+    con = MeshConnection(addrs=addrs,
+                         user=None,
+                         password=None,
+                         socket_timeout=SOCKET_TIMEOUT,
+                         reconnect_max_attempts=1,
+                         reconnect_delay=RECONNECT_DELAY,
+                         connect_now=True,
+                         encoding=ENCODING_DEFAULT)
+    while True:
+        try:
+            s = con.space('test')
+            print s.select()
+        except NetworkError:
+            print 'NetworkError !'
+            break
+        except Exception as e:
+            print e
+        time.sleep(1)
+
+
+# Start replicas
+master = server
+cluster = [master]
```

Comment:
We can use the default instance and one more in the test. No need to start 'dump' default server, I think.